api/cmd/image-loader/main.go

```
@@ -293,6 +293,7 @@ func getTemplateData(versions []*version.MasterVersion, requestedVersion string)
 	fakeCluster.Spec.Version = masterVersion.Version.String()
 	fakeCluster.Spec.ClusterNetwork.Pods.CIDRBlocks = []string{"172.25.0.0/16"}
 	fakeCluster.Spec.ClusterNetwork.Services.CIDRBlocks = []string{"10.10.10.0/24"}
+	fakeCluster.Spec.ClusterNetwork.DNSDomain = "cluster.local"
```

Is this required?

---

I prefer to return an error in `UserClusterDNSPolicyAndConfig` when the DNSDomain is empty. Unfortunately the image-loader fails then, when fakeCluster has no domain.